Flagsmith.Engine/Flagsmith.Engine.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<RootNamespace>Flagsmith_engine</RootNamespace>
```

Comment:
Should this now be `Flagsmith.Engine` or just `Engine`?

Reply:
we can go with  Flagsmith.Engine